.nvmrc

```diff
@@ -0,0 +1 @@
+v18.3.0
```

Comment:
Please use a stable version (an LTS version, any 16.x should work)

Reply:
Fixed, thanks